tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -113,15 +100,17 @@ wavelengths <- wavelengths*1000
 
 Next, we will extract all reflectance values for one pixel. This makes up the 
 spectral signature or profile of the pixel. To do that, we'll use the `h5read` 
-function.
+function. Here we pick an arbitrary pixel at `(100,35)`, and use the `NULL` 
```

Comment:
Don't include if not worth it but what if you wanted to choose a specific pixel or at least a pixel from a specific land cover type.  How can one figure out which pixel to choose instead of a random one?

Reply:
This is what I wanted to get at in the 'advanced-spectral-signatures' tutorial. I think it makes sense to have this information as its own tutorial to explain just the extract and plot process. The next step will be to select specific pixels to explore in the following tutorial.